syntax = "proto3";
package BehaviourLog;

//行为日志信息发送请求
message behaviour_log_req
{
    uint32 log_type = 1;//日志类型
    bytes log_info = 2;//日志信息
}
//行为日志信息发送响应
message behaviour_log_ack
{
    int32 err_no           = 1;
    bytes err_msg          = 2;
}

